__init__.py

```diff
@@ -310,12 +308,34 @@ def execute(self, _context):
         print("Starting Import")
         import webbrowser
         print("Starting Login")
-        if thangs_api.bearer == "":
+
+        # __location__ = os.path.realpath(
+        #     os.path.join(os.getcwd(), os.path.dirname(__file__)))
+        
+        #bearer_DIR = os.path.join(__location__, 'bearer.json')
+        if not os.path.exists('bearer.json'):

```

Comment:
This might put the json file in an inconsistent location, depending on how you launch blender, as that can change the current working directory.

How about we take the same approach that the automatic updater does, and use a path relative to the current file (which should be in the addons folder for blender).

So, let's have the path to the file be 

```
os.path.join(os.path.dirname(__file__), 'bearer.json')
```